### ISSUE TITLE
Formatting of C# Code inside code cell

### ISSUE THREAD

**Issue by vabka** [dotnet/interactive#1666]
**Is your feature request related to a problem? Please describe.**
I want to be able to use code-formatting and refactoring features from other extensions in VS Code
Currently, when i call "Format Document" i get an "There is no formatter for 'dotnet-interactive.csharp' file installed"

**Describe the solution you'd like**
Code cell acting like plain file.

--------
There is an issue from omnisharp-vscode: https://github.com/OmniSharp/omnisharp-vscode/issues/4804

**Comment by heartacker** [comment]
we want it;
and python is supported
![image](https://user-images.githubusercontent.com/1876302/144333654-0edf7188-6a61-440c-a619-59439078d27f.png)

**Comment by mika76** [comment]
This would be great - especially useful when writing lots of linq code

**Comment by mika76** [comment]
I'd like to bump this (@jonsequitur maybe?) - I know it's not some crazy productivity feature or something, but man it would be so useful. I also wonder if it wouldn't be pretty easy to implement just using the C# extensions formatter?

**Comment by jonsequitur** [comment]
I agree it would be very useful. I've added the `help wanted` tag because the team won't have time for it any time soon but we'd be happy to help someone prepare a PR.

**Comment by heartacker** [comment]
https://github.com/microsoft/vscode/issues/212381#issuecomment-2187015054
